lib/rubocop/cop/style/alias.rb

```diff
@@ -27,8 +27,11 @@ def on_send(node)
         def on_alias(node)
           # alias_method can't be used with global variables
           return if node.children.any?(&:gvar_type?)
+          # alias_method can't be used in instance_eval blocks
+          scope_type = scope_type(node)
+          return if @must_use_alias
```

Comment:
Once set, the value of `@must_use_alias` remains and affects the inspection of the next `alias` found in the same file. I think it's better if you add a third scope type called `:instance_eval`.